### PR TITLE
Feature/refactor code 1

### PR DESCRIPTION
**Pull request by dhungelgd**

### CODE REVIEW

File: ramp/core/core.py [hunk 1]
```diff
@@ -60,30 +61,147 @@ def __init__(self, user, number = 1, power = 0, num_windows = 1, func_time = 0,
...
-        self.random_var_1 = int(r_w*np.diff(w1)) #calculate the random variability of window1, i.e. the maximum range of time they can be enlarged or shortened
-        self.random_var_2 = int(r_w*np.diff(w2)) #same as above
-        self.random_var_3 = int(r_w*np.diff(w3)) #same as above
         self.user.App_list.append(self) #automatically appends the appliance to the user's appliance list
 
         if self.activate == 1:
             self.cw11 = self.window_1
             self.cw12 = self.window_2
 
+    def calc_rand_window(self, window_num=1, max_window_range=np.array([0, 1440])):
```
Comment (Bachibouzouk) [inline ramp/core/core.py:83]: I would add a docstring (""" write docstring description""", below the method definition) to explain what the method is doing

File: ramp/core/stochastic_process.py [hunk 2]
```diff
@@ -23,6 +22,13 @@ def calc_peak_time_range(user_list, peak_enlarg):
     rand_peak_enlarge = round(math.fabs(peak_time - random.gauss(peak_time, peak_enlarg * peak_time)))
     return np.arange(peak_time - rand_peak_enlarge , peak_time + rand_peak_enlarge)  # the peak_time is randomly enlarged based on the calibration parameter peak_enlarg
 
+def randomise_variables(var):
```
Comment (Bachibouzouk) [inline ramp/core/stochastic_process.py]: I would add a second argument `norm` which is 1 by default. This way you can pass the value of for example `App.P_11` as norm and perform the multiplication inside the function, thus making the code even slicker

File: ramp/core/stochastic_process.py [hunk 3]
```diff
@@ -48,69 +48,75 @@ def randomise_cycle(App):
             random_cycle3 = random.choice([calc_random_cycle(App.t_31, App.p_31, App.t_32, App.p_32, rand_r_c3),calc_random_cycle(App.t_32, App.p_32, App.t_31, App.p_31, rand_r_c3)])
     return random_cycle1, random_cycle2, random_cycle3
 
-#%% Core model stochastic script
+def generate_user_load(prof_i, Us, peak_time_range, Year_behaviour):
```
Comment (Bachibouzouk) [inline ramp/core/stochastic_process.py]: Why not make this function a method of the class `User`?

File: ramp/core/stochastic_process.py (hunk 2 above)
Comment (dhungelgd) [reply ramp/core/stochastic_process.py]: Could you elaborate this a bit, please?

**Pull request closed without merging**